clients/da-vinci-client/src/main/java/com/linkedin/davinci/config/VeniceStoreVersionConfig.java

```diff
@@ -30,13 +31,15 @@ public class VeniceStoreVersionConfig extends VeniceServerConfig {
 
   private boolean restoreDataPartitions = true;
   private boolean restoreMetadataPartition = true;
+  private boolean blobTransferBatchOnlyEnabled;
 
   public VeniceStoreVersionConfig(
       String storeVersionName,
       VeniceProperties storeProperties,
       Map<String, Map<String, String>> kafkaClusterMap) {
     super(storeProperties, kafkaClusterMap);
     this.storeVersionName = storeVersionName;
+    this.blobTransferBatchOnlyEnabled = storeProperties.getBoolean(ENABLE_BLOB_TRANSFER_FOR_BATCH_ONLY_STORE, false);
```

Comment:
Is it possible that we add the validation somewhere so when the config is enabled for a store that is not compatible, we might want to throw exception.

Reply:
Is this what you're asking - 
If the `ENABLE_BLOB_TRANSFER_FOR_BATCH_ONLY_STORE` is enabled for a different config type, we should throw an error?